superset-frontend/src/components/Button/Button.stories.jsx

```diff
@@ -26,68 +26,54 @@ export default {
 };
 
 export const STYLES = {
-  label: 'Types',
+  label: 'styles',
   options: {
-    Primary: 'primary',
-    Secondary: 'secondary',
-    Tertiary: 'tertiary',
-    Dashed: 'dashed',
-    Danger: 'danger',
-    Warning: 'warning',
-    Success: 'success',
-    Link: 'link',
-    Default: 'default',
-    None: null,
+    primary: 'primary',
+    secondary: 'secondary',
+    tertiary: 'tertiary',
+    dashed: 'dashed',
+    danger: 'danger',
+    warning: 'warning',
+    success: 'success',
+    link: 'link',
+    default: 'default',
+    none: null,
   },
   defaultValue: null,
-  // groupId: 'ButtonType',
 };
 
 export const SIZES = {
-  label: 'Sizes',
+  label: 'sizes',
   options: {
-    XS: 'xsmall',
-    S: 'small',
-    Default: null,
-    L: 'large',
+    xsmall: 'xsmall',
+    small: 'small',
+    default: 'default',
   },
   defaultValue: null,
 };
 
-// TODO remove the use of these class names in the codebase where they're not necessary
-//   'fetch' // haven't yet seen this (in ResultSet.tsx) actually show up to verify the styles are needed
-//   'm-r-3' // open a PR with a prop of `pullRight` that adds an automatic right-margin for second and subseqent sibling buttons.
-
-const TYPES = {
-  label: 'Type',
-  options: {
-    Submit: 'submit',
-    Button: 'button',
-    None: null,
-  },
-  defaultValue: null,
-};
 const TARGETS = {
-  label: 'Target',
+  label: 'target',
   options: {
-    Blank: '_blank',
-    None: null,
+    blank: '_blank',
+    none: null,
   },
   defaultValue: null,
 };
+
 const HREFS = {
-  label: 'HREF',
+  label: 'href',
   options: {
-    Superset: 'http://https://superset.apache.org/',
-    None: null,
+    superset: 'https://superset.apache.org/',
```

Comment:
I have no strong feelings about the capitalization here, but I wonder if we should stick to the Superset caps rules (Sentence case) or if there's a rationale behind switching to lower case.

Reply:
Here I made a conscious change because those are properties and should appear as such in controls panel. It's the same as reading an API. In fact, if we don't specify names for properties, Storybook will automatically create them using  lower case.